src/main/resources/jenkins/branch/MultiBranchProjectEmptyView/main.jelly

```diff
@@ -21,15 +21,23 @@
  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  ~ THE SOFTWARE.
  -->
-
 <?jelly escape-by-default='true'?>
-<st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout title="${%Welcome}">
-    <j:set var="view" value="${it}"/>
-    <!-- expose view to the scripts we include from owner -->
-    <st:include page="sidepanel.jelly"/>
-    <l:main-panel>
-      <st:include page="main.jelly"/>
-    </l:main-panel>
-  </l:layout>
-</st:compress>
+<div xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+    <div style="margin: 20px 0; padding: 10px 30px 20px 30px; background: #f8f8f8; border-radius: 10px; max-width: 55em;">
```

Comment:
[2 space indent in XML](https://github.com/jenkinsci/branch-api-plugin/blob/master/CONTRIBUTING.md#indentation)